cpp/marc_tools/single_marc_convert_keywords.cc

```diff
@@ -36,19 +37,14 @@ namespace {
 
 
 [[noreturn]] void Usage() {
-    std::cerr << "Usage: " << ::progname << " [--quiet] [--limit max_no_of_records] [--output-individual-files] marc_input marc_output [CTLN_1 CTLN_2 .. CTLN_N]\n"
-              << "       Autoconverts the MARC format of \"marc_input\" to \"marc_output\".\n"
-              << "       Supported extensions are \"xml\", \"mrc\", \"marc\" and \"raw\".\n"
-              << "       All extensions except for \"xml\" are assumed to imply MARC-21.\n"
-              << "       If a control number list has been specified only those records will\n"
-              << "       be extracted or converted.\n"
-              << "       If --output-individual-files is specified marc_output must be a writable directory\n"
-              << "       and files are named from the control numbers and written as XML\n\n";
-    std::exit(EXIT_FAILURE);
+    ::Usage(" gnd_input keyword_input keyword_matches_output keyword_without_matches_output \n"
```

Comment:
The leading space is not necessary here.  Also there should be only one newline at the end because the library function already adds a newline.